core/src/main/java/org/axonframework/commandhandling/model/AbstractRepository.java

```diff
@@ -284,14 +288,21 @@ protected void postDelete(A aggregate) {
 
     @Override
     public void send(Message<?> message, ScopeDescriptor scopeDescription) throws Exception {
-        if (scopeDescription instanceof AggregateDescriptor) {
-            load(((AggregateDescriptor) scopeDescription).getIdentifier().toString()).handle(message);
+        if (canResolve(scopeDescription)) {
+            String aggregateIdentifier = ((AggregateScopeDescriptor) scopeDescription).getIdentifier().toString();
+            A aggregate = load(aggregateIdentifier);
+            if (aggregate != null) {
```

Comment:
According to the documentation, when an aggregate isn't found, an exception is thrown.
Please ensure there is a test that verifies messages for an "unresolvable scope" are ignored. For example, a scope that describes an aggregate that doesn't exist.